packages/amplify_flutter/android/src/test/kotlin/com/amazonaws/amplify/AmplifyTest.kt

```diff
@@ -13,15 +13,15 @@ import org.robolectric.RobolectricTestRunner
 
 
 @RunWith(RobolectricTestRunner::class)
-class AmplifyCoreTest {
+class AmplifyTest {
 
-  lateinit var plugin: Core
+  lateinit var plugin: Amplify
```

Comment:
Should probably change the variable name here, too. `amplify`? `fascade`?
```suggestion
  lateinit var fascade: Amplify
```